recipes/pip-audit/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "pip-audit" %}
+{% set version = "1.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pip-audit-{{ version }}.tar.gz
+  sha256: 61d772968b6ef644f43ecceace89665d28d7ea521a9390e59188c4189d580856
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  entry_points:
+    - pip-audit = pip_audit._cli:audit
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - cachecontrol >=0.12.10
+    - cyclonedx-python-lib >=0.11.1
+    - dataclasses >=0.6  # [py<37]
```

```suggestion
    - dataclasses >=0.6
```
COnda forge maintains empty packages for certain python versions: https://conda-forge.org/docs/maintainer/knowledge_base.html#empty-python-packages

---

Thank you. I could not recall exactly where I had seen a reference to dataclasses. :) 

Just made the changes.